src/components/TabsMovies/TabsMovies.jsx

```diff
@@ -18,7 +22,9 @@ export const TabsMovies = () => {
       {tabsContent.map(({ name, tabId }) => {
```

Comment:
Lets move `tabsContent` variable outside of component or maybe somwhere in a file with mocked values